cluster/gce/gci/configure-helper.sh

```diff
@@ -1655,24 +1650,20 @@ function prepare-konnectivity-server-manifest {
   params+=("--log-file=/var/log/konnectivity-server.log")
   params+=("--logtostderr=false")
   params+=("--log-file-max-size=0")
-  params+=("--server-ca-cert=${KONNECTIVITY_SERVER_CA_CERT_PATH}")
-  params+=("--server-cert=${KONNECTIVITY_SERVER_CERT_PATH}")
-  params+=("--server-key=${KONNECTIVITY_SERVER_KEY_PATH}")
-  params+=("--cluster-ca-cert=${KONNECTIVITY_AGENT_CA_CERT_PATH}")
-  params+=("--cluster-cert=${KONNECTIVITY_AGENT_CERT_PATH}")
-  params+=("--cluster-key=${KONNECTIVITY_AGENT_KEY_PATH}")
+  params+=("--udsName=/etc/srv/kubernetes/konnectivity/konnectivity-server.socket")
```

Comment:
it might be worse to cut a PR in apiserver-network-proxy first for renaming udsName to uds-name

Reply:
I'll wait for that PR and your token authentication PR to merge and update this.